src/main/java/com/zontik/groshiky/controllers/BaseController.java

```diff
@@ -0,0 +1,12 @@
+package com.zontik.groshiky.controllers;
+
+import com.zontik.groshiky.model.User;
+import org.springframework.security.core.context.SecurityContextHolder;
+
+
+public abstract class BaseController {
+    protected Integer getUserId() {
```

Comment:
It is better to have method that returns the current (authenticated user) instead of ID or something like this, isn't it? In that case you don't need any additional logic in your controller with user services.

Reply:
Yes. But putting current User entity is not the best solution I think. Next time we will add special entity, that will contain all user details, reqired for authentication. Anyway, security context should not store User entity as it presented in DB. So for now it is OK I think :) But should be updated in a future